aws-event-bridge/src/main/scala/akka/stream/alpakka/aws/eventbridge/javadsl/EventBridgePublisher.scala

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2016-2019 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.aws.eventbridge.javadsl
+
+import java.util.concurrent.CompletionStage
+
+import akka.stream.alpakka.aws.eventbridge.EventBridgePublishSettings
+import akka.stream.javadsl.{Flow, Keep, Sink}
+import akka.{Done, NotUsed}
+import software.amazon.awssdk.services.eventbridge.EventBridgeAsyncClient
+import software.amazon.awssdk.services.eventbridge.model._
+
+/**
+ * Java API
+ * Amazon EventBridge publisher factory.
+ */
+object EventBridgePublisher {
```

Comment:
I see that you're following the factory method naming convention from the SNS Java API, but I suggest dropping the `create` prefix from all these methods so they match the Scala API.